### DISCUSSION TITLE
Can the window size be fixed and unchanged？

### DISCUSSION THREAD

**Discussion by mrhanCH** [zauberzeug/nicegui#2243] [Q&A]
```py
ui.run(reload=False, port=native.find_open_port(), native=True, window_size=(1280, 720))
```

I used Native mode for packaging and set the window size using "window_size" to package and successfully run the program. However, I want the program window size to be fixed and users cannot drag and drop to adjust the window size. Just like the setFixedSize method in Pyqt5. May I ask if it can be achieved?

**Comment by falkoschindler** [comment]
You can set additional arguments for native mode using `app.native.start_args` and `app.native.window_args`. Here is an example how to stop the window from being resizable:

```py
from nicegui import app, ui

app.native.window_args['resizable'] = False

ui.run(native=True, window_size=(1280, 720))
```

**Reply by mrhanCH** [reply]
It was my mistake, I saw it written in the document, thank you very much!
